packages/logger-winston/src/index.ts

```
@@ -0,0 +1,99 @@
+    if (this.winstonLogger.isErrorEnabled()) {
+      this.winstonLogger.error(...prepareArgs(args));
+    }
+  }
+  debug(...lazyArgs: LazyLoggerMessage[]) {
+    if (this.winstonLogger.isDebugEnabled()) {
+      this.winstonLogger.debug(...prepareArgs(lazyArgs));
+    }
+  }
+  child(nameOrMeta: string | Record<string, string | number>) {
+    if (typeof nameOrMeta === 'string') {
+      nameOrMeta = { name: this.name ? this.name.includes(nameOrMeta) ? this.name : `${this.name}, ${nameOrMeta}` : nameOrMeta };
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)